Source: Gentoo (04_mpl-g-4.3-fix.bug-1528-not-included.patch)
Upstream: ?
Reason: Fix GCC-4.3* compile.


From 902912000f991b385ba063496147cac4ba7b55c2 Mon Sep 17 00:00:00 2001
From: Tim Blechmann <tim@klingt.org>
Date: Sat, 2 Feb 2008 17:15:32 +0100
Subject: [PATCH] mpl: g++-4.3 fix

---
 boost/mpl/zip_view.hpp |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/boost/mpl/zip_view.hpp b/boost/mpl/zip_view.hpp
index ee13d03..98a0c7a 100644
--- a/boost/mpl/zip_view.hpp
+++ b/boost/mpl/zip_view.hpp
@@ -37,7 +37,7 @@ struct zip_iterator
     typedef zip_iterator<
           typename transform1<
                 IteratorSeq
-              , next<_1>
+              , boost::mpl::next<_1>
             >::type
         > next;
 };
@@ -48,8 +48,8 @@ template<
 struct zip_view
 {
  private:
-    typedef typename transform1< Sequences, begin<_1> >::type first_ones_;
-    typedef typename transform1< Sequences, end<_1> >::type last_ones_;
+    typedef typename transform1< Sequences, boost::mpl::begin<_1> >::type first_ones_;
+    typedef typename transform1< Sequences, boost::mpl::end<_1> >::type last_ones_;
     
  public:
     typedef nested_begin_end_tag tag;
-- 
1.5.3.8

